cms/envs/production.py

```diff
@@ -357,11 +357,12 @@
 AWS_BUCKET_ACL = AWS_DEFAULT_ACL
 AWS_QUERYSTRING_EXPIRE = 7 * 24 * 60 * 60  # 7 days
 AWS_S3_CUSTOM_DOMAIN = AUTH_TOKENS.get('AWS_S3_CUSTOM_DOMAIN', 'edxuploads.s3.amazonaws.com')
+AWS_S3_ENCRYPTION = False
```

Comment:
This needs to be configurable using the `ENV_TOKENS`.
```suggestion
AWS_S3_ENCRYPTION = ENV_TOKENS.get('AWS_S3_ENCRYPTION', False)
```

Reply:
Done